ARDroneLib/VP_SDK/VP_Os/linux/intrin.h

```diff
@@ -9,22 +9,37 @@
 
 #if TARGET_CPU_X86 == 1
 
-static INLINE uint32_t _BitScanReverse(uint32_t* index, uint32_t mask)
+/*static INLINE uint32_t _BitScanReverse(uint32_t* index, uint32_t mask)
```

Comment:
Is this necessary? I thought for x86 platforms, there should be no change. Apparently this is [breaking the build on Travis](https://travis-ci.org/AutonomyLab/ardronelib/builds/144607148).